xchange-deribit/src/main/java/org/knowm/xchange/deribit/v2/DeribitErrorAdapter.java

```diff
@@ -1,40 +0,0 @@
-package org.knowm.xchange.deribit.v2;
-
-import static org.apache.commons.lang3.StringUtils.isNotEmpty;
-
-import org.knowm.xchange.deribit.v2.dto.DeribitError;
-import org.knowm.xchange.deribit.v2.dto.DeribitException;
-import org.knowm.xchange.exceptions.CurrencyPairNotValidException;
-import org.knowm.xchange.exceptions.ExchangeException;
-
-public class DeribitErrorAdapter {
```

Comment:
restore this class

this would be a step back from our best practices: https://github.com/knowm/XChange/wiki/New-Implementation-Best-Practices#step-7

we are loosing functionality here (CurrencyPairNotValidException detection) and closing the path to more specific error detection